jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ArtifactHandler.kt

```diff
@@ -159,7 +159,8 @@ class ArtifactHandler(private val project: Project, private val clientAdaptor: G
                 null,
                 null,
                 null,
-                false
+                false,
+                jobId
             )
             dialog.isModal = true
             dialog.showAndGet()
```

Comment:
since we don't own this dialog, it's probably cleaner to just emit telemetry here instead of subclassing and having to override the entire constructor

```kt
CodetransformTelemetry.vcsDiffViewerVisible

if (dialog.showAndGet()) {
    // true means OK_EXIT_CODE
    CodetransformTelemetry.vcsViewerSubmitted
} else {
    CodetransformTelemetry.vcsViewerCanceled
}
```

Reply:
Thanks for the tips! I'll change it to this